ni/measurementlink/measurement/v2/measurement_service.proto

```diff
@@ -124,11 +124,11 @@ message ConfigurationParameter {
   // Well-known annotations:
   // - Type specialization. The keys to other annotations will be read based on the value of `ni/type_specialization` annotation.
   //   - Key: "ni/type_specialization"
-  //   - Common Values: "pin", "path", "enum", ...
-  //   - "pin" and "path" parameters require the type field to be TYPE_STRING.
+  //   - Common Values: "ioresource", "path", "enum"...
+  //   - "ioresource" and "path" parameters require the type field to be TYPE_STRING.
   //   - "enum" parameters requires the type field to be TYPE_ENUM.
-  // - For string parameter with ni/type_specialization annotation equals "pin"
-  //   - Key: "ni/pin.instrument_type"
+  // - For string parameter with ni/type_specialization annotation equals "ioresource"
+  //   - Key: "ni/ioresource.instrument_type"
```

Comment:
We will also plan to add annotations for "ni/ioresource.io_type" and some key/format yet to be finalized for filter parameters to the I/O Discovery Service. The immediate need for the latter would be for specifying the channel type for DAQmx (AI, AO, etc.). You can either do the work to figure that out now or add them in follow on PRs.  

For the `instrument_type` annotation, it might also be nice to add a little more documentation about how it is used. Something to the effect that it is used by UIs to limit selection of I/O to the specified instrument type would be fine.

Reply:
For the other annotations I've created task AB#2719576.

I've added a bit more description of the `instrument_type` annotation as suggested.